common/changes/@bentley/itwin-connector-framework/connector-fwk-rename_2021-06-17-20-15.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/itwin-connector-framework",
+      "comment": "renamed imodel-bridge to itwin-connector-framework and changed \"bridge\" symbols to \"connector\".",
+      "type": "none"
+    }
+  ],
+  "packageName": "@bentley/itwin-connector-framework",
+  "email": "zachzhu2016@gmail.com"
```

Comment:
you may want to switch to using the noreply email address rather than your gmail